frontend/src/editor/player.tsx

```diff
@@ -12,6 +12,12 @@ import { Editor } from 'slate';
 import { useButtonHoldRepeat } from '../utils/button_hooks';
 import { useLocalStorage } from '../utils/use_local_storage';
 
+const DOUBLE_TAP_THRESHOLD_MS = 250;
+const SKIP_BUTTON_SEC = 2;
+const SKIP_SHORTCUT_SEC = 3;
```

Comment:
Why do we use different values here?

Reply:
Because it feels better for me. Do you want to try it and see what values work for you the best?

Reply:
I'm fine with this, just wanted to know if there is a deeper reason